resources/svgicons/dock-sticky.svg

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Too thin of a border in unscaled version.

![image](https://user-images.githubusercontent.com/9251300/53495877-1bad5200-3aa1-11e9-86ee-d6c0ce1938c8.png)